gub/specs/util-linux.py

```diff
@@ -1,11 +1,52 @@
 #
 from gub import tools
+from gub import target
 
 class Util_linux__tools (tools.AutoBuild):
-    source = 'http://www.nl.kernel.org/pub/linux/utils/util-linux-ng/v2.16/util-linux-ng-2.16.tar.gz'
-    dependencies = ['libtool']
+    source = 'https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.16/util-linux-ng-2.16.tar.gz'
+    dependencies = ['libtool', 'bash']
+    patches = ['util-linux-2.16-sysmacro.patch']
     configure_flags = (tools.AutoBuild.configure_flags
                 + ' --disable-tls'
                 + ' --disable-makeinstall-chown'
                 + ' SHELL=%(tools_prefix)s/bin/bash'
                 + ''' CFLAGS='-DLINE_MAX=1024' ''')
+
+class Util_linux (target.AutoBuild):
+    # util-linux-2.23 requires O_CLOEXEC but our glibc does not have it.
+    source = 'https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.22/util-linux-2.22.2.tar.xz'
```

Comment:
Just curious: Do we really need both 'util-linux' versions 2.16 and 2.22.2?

Reply:
util-linux-2.22.2 could not compile for tools::util-linux.

Reply:
OK.  Please add a remark!

Reply:
done. Thanks.